uvicorn/lifespan/auto.py

```diff
@@ -6,6 +6,9 @@ def LifespanAuto(config):
     if not config.loaded:
         config.load()
 
+    if not config.loop_setup:
+        config.setup_event_loop()
```

Comment:
I'm not sure if we need these anymore - may be that we can get away with _only_ making the check in the server?